packages/rocketchat-videobridge/client/views/videoFlexTab.js

```diff
@@ -0,0 +1,73 @@
+/* globals JitsiMeetExternalAPI */
+
+Template.videoFlexTab.helpers({
+
+});
+
+Template.videoFlexTab.onCreated(function() {
+	let rid = this.data && this.data.rid;
+	let api;
+
+	let timeOut = null;
+
+	let domain = RocketChat.settings.get('Jitsi_Domain');
+	let jitsiRoom = CryptoJS.MD5(RocketChat.settings.get('uniqueID') + rid).toString();
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [A function with a name starting with an uppercase letter should only be used as a constructor.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2718060148/issues/source?bid=3199634&fileBranchId=3342079#l14)